src/components/career-form/career-file-input.tsx

```diff
@@ -0,0 +1,105 @@
+import { Input, InputContainer } from '../form/controls';
+import React, { ReactNode, useCallback } from 'react';
+import { DropzoneRootProps, useDropzone } from 'react-dropzone';
+import styled from 'styled-components';
+import { theme } from '../layout/theme';
+import {
+  FieldValues,
+  UseFormClearErrors,
+  UseFormRegister,
+  UseFormSetValue,
+} from 'react-hook-form';
+
+interface FileInputProps {
+  setValue: UseFormSetValue<FieldValues>;
+  clearErrors: UseFormClearErrors<FieldValues>;
+  register: UseFormRegister<FieldValues>;
+  name: string;
+  selectedFiles: FileList;
+  error: boolean;
+  children: ReactNode | ReactNode[];
+}
+
+export const FileInput = ({
+  setValue,
+  clearErrors,
+  name,
+  register,
+  selectedFiles,
+  error,
+  children,
+}: FileInputProps) => {
+  const onDrop = useCallback(
+    (droppedFiles) => {
+      // dropped files are appended to already selected files
+      const dataTransfer = new DataTransfer();
+      if (selectedFiles) {
+        for (let i = 0; i < selectedFiles.length; i++)
+          dataTransfer.items.add(selectedFiles[i]);
+      }
+      for (let i = 0; i < droppedFiles.length; i++)
+        dataTransfer.items.add(droppedFiles[i]);
+      setValue(name, dataTransfer.files, { shouldDirty: true });
+      clearErrors(name);
+    },
+    [selectedFiles, setValue],
+  );
+
+  const { getRootProps, getInputProps } = useDropzone({
+    onDrop,
+    accept: '.pdf',
+  });
+
+  return (
+    <InputContainer>
```

Comment:
Wir brauchen noch ein hover Style, wenn man die Datei in den Browser zieht. Damit der User weiß, wo der drop gemacht werden muss. So wie hier: https://react-dropzone.js.org/#!/Styling%20Dropzone

Reply:
Am besten kurz mit Verena checken, ob die dazu noch ein Design hinzufügen kann

Reply:
Ich hab ein isDragActive bei dem Feld eingefügt. Das Styling ist bisher nur ein Beispiel